### DISCUSSION TITLE
Timeframe on ASR finetuning scripts getting added as a new tag?

### DISCUSSION THREAD

**Discussion by Jesteinbe** [NVIDIA-NeMo/NeMo#7502] [Q&A] [answered]
I've recently started using Nemo and for development purposes just chose the most recent tag (1.20.0) so that I could track a relatively stable release of Nemo. I put my own tools together to finetune ASR models but then came across the new tools you pushed a month or so ago based on https://github.com/NVIDIA/NeMo/pull/7263. Is there a rouch time frame for when that code will make it into the next tag? In general, how frequent are new tags added? Any info would be helpful!

**Comment by titu1994** [comment, marked as answer]
We used to do monthly releases, but the number of things we developed and tested on one month cadence is limited so we have moved to a two month cadence for now. The next release is 1.21, to be released by the beginning of next month.

Also, you do not need to wait for a pip release, you can install a mostly stable version from the git branch labeled r1.21.0 (which is the release candidate)

**Reply by Jesteinbe** [reply]
Perfect thanks! I'm pretty conservative when it comes to these types of things (not just for Nemo) so I often choose 1-2 releases prior to whatever is current so that i'm mostly up to date with a given package but less susceptible to any of the common issues you might hit using the newest release of a tool. This lets me know that the stable release will happen soon though so i'll at least start playing around with the new code. Thanks again for the quick response!
